travis_pypi_setup.py

```diff
@@ -2,17 +2,18 @@
 # -*- coding: utf-8 -*-
 """Update encrypted deploy password in Travis config file
 """
-
-
 from __future__ import print_function
+
+import argparse
 import base64
 import json
 import os
 from getpass import getpass
+
 import yaml
-from cryptography.hazmat.primitives.serialization import load_pem_public_key
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives.asymmetric.padding import PKCS1v15
+from cryptography.hazmat.primitives.serialization import load_pem_public_key
```

Comment:
`prospector: pylint: Unable to import 'cryptography.hazmat.primitives.serialization' (import-error)`